core/modules/system/css/menu-dropdown.theme.breakpoint-queries.css

```diff
@@ -0,0 +1,160 @@
+/* Switch to desktop layout.
+-----------------------------------------------
+   These transform the menu tree from
+   collapsible to desktop (navbar + dropdowns)
+-----------------------------------------------*/
+/* start... (it's not recommended editing these rules) */
```

Comment:
Could this be made to match other Start/End blocks?

Reply:
@quicksketch Made a commit to address this. Let me know if it still needs work.